arm-dependencies.sls

```diff
@@ -0,0 +1,104 @@
+{% from 'common/map.jinja' import config %}
+
+arm-dependencies:
+  pkg.installed:
+    - pkgs:
+      - g++-aarch64-linux-gnu
+      - g++-arm-linux-gnueabihf
+
+{% set rootfs_repo = 'https://servo-rust.s3.amazonaws.com/ARM' %}
+{% set targets = [{
+                    'name': 'arm-linux-gnueabihf',
+                    'symlink_name': 'arm-unknown-linux-gnueabihf',
+                    'version': 'v1',
+                    'local_name': 'armhf-trusty-libs.tgz',
+                    'hash': 'd9a31ed488e4f848efcd07f71aa167fc73252da2a2c3b53ba8216100e2b4302b5ccd273b27c434ad189650652a1877607d328ff6b8e1edb5aa68a8927c617b49',
+                  },
+                  {
+                    'name': 'aarch64-linux-gnu',
+                    'symlink_name': 'aarch64-unknown-linux-gnu',
+                    'version': 'v1',
+                    'local_name': 'arm64-trusty-libs.tgz',
+                    'hash': '6c86097188b70940835b2fc936fe70f01890fae45ba4ef79dcccc6a552ad319dcba23e21d6c849fd9d396e0c2f4476a21c93f9b3d4abb4f34d69f22d18017b1b',
+                  }] %}
+
+{% set binaries = [
```

Comment:
We're getting some broken symlinks: `gcov-tool` for arm32 and arm64, as well as `dwp` and `ld.gold` on arm64. If we don't need these, we should remove them from the list.